volume_types.go

```diff
@@ -375,3 +385,33 @@ func (v *Volume) Fields() []string {
 func (n *ApplianceInstance) Fields() []string {
 	return []string{"id", "name", "service_tag"}
 }
+
+// MetroConfig defines the properties required to configure a metro volume replication session.
+type MetroConfig struct {
+	// RemoteSystemID is a required parameter specifying the remote PowerStore array/cluster on which
+	// the metro volume should be replicated.
+	RemoteSystemID string `json:"remote_system_id"`
+	// RemoteApplianceID is an optional parameter specifying a specific remote PowerStore appliance
+	// on which the metro volume or volume group should be replicated.
+	RemoteApplianceID string `json:"remote_appliance_id,omitempty"`
+}
+
+// MetroSessionID id the Unique identifier of the replication session assigned
+// to the volume if it has been configured as a metro volume between two PowerStore clusters.
+type MetroSessionID struct {
```

Comment:
Should this be MetroSessionResponse since this has ID field in it?

Reply:
also make sense to me